app/lib/grades/pages/create_term_page/create_term_page.dart

```diff
@@ -136,6 +140,63 @@ class _NameField extends StatelessWidget {
   }
 }
 
+class _GradingSystem extends StatelessWidget {
+  const _GradingSystem();
+
+  @override
+  Widget build(BuildContext context) {
+    final gradingSystem =
+        context.select<CreateTermPageController, GradingSystem>(
+            (controller) => controller.view.gradingSystem);
+    return InkWell(
+      borderRadius: BorderRadius.circular(12),
+      onTap: () async {
+        final res = await showDialog<GradingSystem?>(
+          context: context,
+          builder: (context) => SimpleDialog(
+            title: const Text("Notensystem auswählen"),
+            children: [
+              for (final gradingSystem in GradingSystem.values)
+                ListTile(
+                  title: Text(gradingSystem.displayName),
+                  onTap: () {
+                    Navigator.of(context).pop<GradingSystem?>(gradingSystem);
+                  },
+                ),
+            ],
+          ),
+        );
```

Comment:
Later, this dialog will be replaced with a shared one which is also used on grades dialog page (see #1456).